public/src/main/java/com/revenuecat/purchases/Offering.kt

```diff
@@ -15,6 +15,7 @@ package com.revenuecat.purchases
 data class Offering constructor(
```

Comment:
Also, we need to add the documentation for that property this data class above right?